tests/test_regrid.py

```diff
@@ -39,7 +40,63 @@ def sample_grid_ds():
         ("nearest", CDO_DATA["nearest"]),
     ],
 )
-def test_regridder(sample_input_data, sample_grid_ds, method, cdo_file):
-    ds_regrid = sample_input_data.regrid.regrid(sample_grid_ds, method=method)
+def test_basic_regridders_ds(sample_input_data, sample_grid_ds, method, cdo_file):
+    """Test the dataset regridders (except conservative)."""
+    regridder = getattr(sample_input_data.regrid, method)
+    ds_regrid = regridder(sample_grid_ds)
     ds_cdo = xr.open_dataset(cdo_file)
     xr.testing.assert_allclose(ds_regrid.compute(), ds_cdo.compute())
+
+
+@pytest.mark.parametrize(
+    "method, cdo_file",
+    [
+        ("linear", CDO_DATA["linear"]),
+        ("nearest", CDO_DATA["nearest"]),
+    ],
```

Comment:
Of course it would be nice to add check for `cubic` as well since we have it implemented. But at this point, without proper testing data for it. It is fine to simply ignore it here.